packages/frontend/web/pages/Article.tsx

```diff
@@ -81,36 +81,44 @@ const headerAdWrapperHidden = css`
     display: none;
 `;
 
-const headerAd = css`
+const adSlotAboveNav = css`
     margin: 0 auto;
     height: 151px;
     padding-bottom: 18px;
     padding-top: 18px;
     text-align: left;
     display: table;
-
     border-bottom: 0.0625rem solid ${palette.neutral[86]};
-
     width: 728px;
 `;
 
-// These are by selector as for dynamically-created ads
-const bodyAdStyles = css`
+const adSlot300px = css`
     .ad-slot {
         width: 300px;
         margin: 12px auto;
         min-width: 300px;
         min-height: 274px;
         text-align: center;
     }
+`;
+
+const adSlotUnspecifiedWidth = css`
+    .ad-slot {
+        margin: 12px auto;
+        min-width: 300px;
+        min-height: 274px;
+        text-align: center;
+    }
+`;
 
+const articleBodyAdStyles = css`
+    ${adSlot300px}
```

Comment:
I think it would make more sense to add the class to both elements, rather than import the css into the block. It's a minor and probably applies to the codebase as a whole, and a convention we should look at!

Reply:
Oh..... Very good suggestion. I was trying to "reuse code", but in this case you are right (and I prefer it the way you suggest).